### PR TITLE
i18n(ko-KR): update `install-and-setup.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `install-and-setup.mdx`
- #9240
- #10144

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10158 is merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
